components/AiTaskInputForm/index.js

```diff
@@ -98,6 +151,8 @@ export default function AiTaskInputForm({
 // Styled components for the form layout
```

Comment:
Leaving comments inside your code when merging is bad practice. You can leave it in for now, but keep it in mind at the end of your project :)

Reply:
Which comments do you mean?